scopes/react/react/templates/workspace-common/files/workspace-config.ts

```diff
@@ -30,7 +31,15 @@ export async function workspaceConfig({ name, defaultScope }: WorkspaceContext)
     'teambit.workspace/variants': parse(`{}`),
   };
 
-  const configMerged = assign(configParsed, variants);
+  const extensions = {
+    [`${defaultScope}/apps/my-app`]: parse(`{}`),
+  };
+
+  let configMerged = assign(configParsed, variants);
+
+  if (isReactApp && defaultScope) {
+    configMerged = assign(configMerged, extensions);
+  }
```

Comment:
This is not the way to do it.
the common file shouldn't be aware of a specific template. never.
please pass to this function another arg like extraConfig, also to `generateFiles` function.
Then inside the `workspaceConfig` function, merge the extra config into the final config.
So it will be controlled by the template itself.